scripts/selenium-test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+setup_service xvfb
+setup_service google-chrome-stable 83.0.4103.61-1
+
+cd /root/okta/okta-angular
+
+get_secret prod/devex/SIWTestClientId clientId
+get_secret prod/devex/SIWTestOktaDomain yourOktaDomain
+echo "export const environment = {production: false,clientId: '$clientId',yourOktaDomain:'$yourOktaDomain'};" > environment.ts
+cat environment.ts
+
+docker-compose up -d
+sleep 120
+
+cd /root/okta/okta-angular/test/selenium-test/selenium
+npm install selenium-webdriver
+wget https://chromedriver.storage.googleapis.com/83.0.4103.39/chromedriver_linux64.zip
```

Comment:
Update this to use chromedriver 87 once upgraded to latest chrome

Reply:
@vijetmahabaleshwar-okta 
Chromedriver 87 was not supported chrome 85. That's why Chromedriver was updated to 85.0.4183.87